src/users/__tests__/userHelpers-test.js

```diff
@@ -180,24 +217,78 @@ describe('sortUserList', () => {
       { full_name: 'Bob', email: 'bob@example.com' },
       { full_name: 'Rick', email: 'rick@example.com' },
     ]);
+
     const presences = {
-      'mark@example.com': { aggregated: { status: 'offline' } },
+      'mark@example.com': {
+        aggregated: { status: 'offline', client: 'ZulipMobile', timestamp: Date.now() },
+        ZulipMobile: {
+          status: 'offline',
+          client: 'ZulipMobile',
+          timestamp: Date.now(),
+          pushable: false,
```

Comment:
`pushable` can't cleanly be supplied, and probably isn't needed anyway. See the comments at its declaration.